app/scripts/modules/appengine/src/serverGroup/configure/wizard/ConfigFileArtifactList.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+
+import { IArtifact, IExpectedArtifact, IPipeline, IStage, StageArtifactSelectorDelegate } from '@spinnaker/core';
+
+interface IConfigFileArtifactListProps {
+  configArtifacts: any[];
+  pipeline: IPipeline;
+  stage: IStage;
+  updateConfigArtifacts: (configArtifacts: any[]) => void;
+  updatePipeline: (changes: Partial<IPipeline>) => void;
+}
+
+export const ConfigFileArtifactList = (props: IConfigFileArtifactListProps) => {
+  const addConfigArtifact = () => {
+    const newConfigArtifacts = [...props.configArtifacts];
+    newConfigArtifacts.push({});
+    props.updateConfigArtifacts(newConfigArtifacts);
```

Comment:
feel free to ignore this suggestion but could probably do this in one line with `concat`, which returns a new array
```suggestion
    props.updateConfigArtifacts(props.configArtifacts.concat([{}]));
```

Reply:
👍